data/entities.yaml

```diff
@@ -1193,3 +1193,24 @@
     inner workings.
   properties: [portable]
   capabilities: [debug]
+
+- name: victrola
+  display:
+    attr: device
+    char: 'Q'
+  description:
+  - |
+    A device for reading and writing data on circular platters made of
+    a soft plastic material, and also, for some reason, transmitting
+    the data as pressure waves through the air.  The stylus must be
+    made of an extremely hard and durable material.
```

Comment:
Or, how about a [quantum dot](https://en.wikipedia.org/wiki/Quantum_dot)?  Sounds sufficiently lofty for an end-game item?

Reply:
I still like `victrola`: it lets you "play (with) records".  And "records" in the audio sense are actually a nice analogue for records in the programming sense: both let you store multiple pieces of data in one object.  In contrast a dot lets you project out fields from a record, but why would it let you *create* records?

What if I just get rid of the description text that mentions "pressure waves through the air"?  I agree it's probably distracting.

Reply:
> the convenience of records might be considered to be their "dot syntax"

The convenience of records is the ability to put multiple pieces of data together into one, where the pieces of data are *named* instead of identified by position (as they would be with a normal product type).  If you still don't like `victrola`, maybe we could consider something related to (name)tags?  Just brainstorming here...

Reply:
I can live with it.  Given that projection is nonetheless an integral operation to record usage, I wonder if we can still work `quantum dot` in somehow.  How about as an ingredient to the `victrola` recipe?

Another consideration is generic vs. trademarked name of the record player.  I take it that `victrola` has become a somewhat genericized brand name, as "kleenex" has become for facial tissues.  There's a whole [Wikipedia section on terminology](https://en.wikipedia.org/wiki/Phonograph#Terminology), but if we want to aim for the least ambiguous, brand-neutral term for the device, maybe we would choose `turntable`.

Reply:
> I wonder if we can still work `quantum dot` in somehow. How about as an ingredient to the `victrola` recipe?

Sounds fun to me.  What should be the recipe for a `quantum dot`?



> I take it that `victrola` has become a somewhat genericized brand name, as "kleenex" has become for facial tissues. There's a whole [Wikipedia section on terminology](https://en.wikipedia.org/wiki/Phonograph#Terminology), but if we want to aim for the least ambiguous, brand-neutral term for the device, maybe we would choose `turntable`.

Sure, I'm aware that `victrola` is a genericized brand name.  Why would it be better to use an unambiguous, brand-neutral term?  To me, `victrola` sounds a lot funnier than `turntable`.

Reply:
@kostmo I added a recipe for `quantum dot` (the Wikipedia page tells me one way to make quantum dots is with silicon with nonthermal plasma, which is created by passing an electric current through a gas; and the resulting quantum dots are often suspended in e.g. water; hence the recipe I came up with), and I added `quantum dot` to the `victrola` recipe and updated the description.  Let me know what you think.

Reply:
Looks good.